components/StoryCards/StoryCard.tsx

```diff
@@ -48,13 +49,21 @@ function StoryCard({ story, size = 'small' }: Props) {
                         />
                     </div>
                 )}
-                <HeadingTag className={styles.title}>
+
+                <HeadingTag
+                    className={classNames(
+                        styles.title,
+                        !hasCategories && styles.noCategories,
+                        !showDate && styles.noDate,
+                        !hasCategories && !showDate && styles.noDateAndCategories,
```

Comment:
Should we use this style instead to keep it consistent with other parts of the codebase

```suggestion
                       {
                          [styles.noCategories]: !hasCategories,
                          [styles.noDate]: !showDate,
                          [styles.noDateAndCategories]: !hasCategories && !showDate
                       }
```

Reply:
Good point 👍